### DISCUSSION TITLE
Real website preview showing unpublished content?

### DISCUSSION THREAD

**Discussion by akno77** [pimcore/pimcore#12140] [Ideas]
As far as I know it is currently not possible to have a real preview of the website showing all unpublished content, right?

It would be great to have sth. like to. Especially if you build up a new website or a new structure with a lot of unpublished pages that should not appear live. But you still want the navigation, links and so on appear in the preview like it would be. Otherwise you don't have a real preview of pages...

**Comment by dvesh3** [comment]
Maybe a nice improvement for **seemode** option in admin, which allows to navigate through links like a real website and you have the option to open any page in editmode but it uses the published version for page previews.

![image](https://user-images.githubusercontent.com/5137917/168843586-c1e80fed-c017-42e4-8055-9fe0d1dfc754.png)
